js/templates/listingCard.html

```diff
@@ -42,7 +42,7 @@
   </div>
   <% if (ob.vendor) { %>
   <a class="userIconWrapper js-userName">
-    <% const vendorTip = `${ob.vendor.name}${ob.vendor.location ? `  📍  ${ob.vendor.location}` : ''}`;
+    <% const vendorTip = `${ob.vendor.name}${ob.vendor.location ? ` ${ob.vendor.location}` : ''}`;
```

Comment:
I think the location should stay, I think we were only talking about removing the icon. @morebrownies which one were you saying?